editor/src/components/navigator/navigator-map.spec.browser2.tsx

```diff
@@ -86,52 +87,82 @@ describe('maps in the navigator', () => {
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: 1,
+        overrideSuccess: true,
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: 2,
+        overrideSuccess: true,
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: 3,
+        overrideSuccess: true,
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: 4,
+        overrideSuccess: true,
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: 4,
+        overrideSuccess: true,
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: -1,
+        overrideSuccess: false,
+      },
+      {
+        arr: [0, 1, 2, 3],
+        overrideCount: 5,
+        overrideSuccess: false,
+      },
+    ]
 
-      testData.forEach(({ arr, overrideCount }) => {
-        it(`shows counter for map items with list length ${arr.length}`, async () => {
-          const renderResult = await renderTestEditorWithCode(
-            getProjectCode(arr, overrideCount),
-            'await-first-dom-report',
-          )
+    testData.forEach(({ arr, overrideCount, overrideSuccess }) => {
+      it(`shows overridden counter ${overrideCount} for map items with list length ${arr.length}`, async () => {
+        const renderResult = await renderTestEditorWithCode(
+          getProjectCode(arr, overrideCount),
+          'await-first-dom-report',
+        )
 
-          const mapParent = EP.fromString('utopia-storyboard-uid/scene-aaa/containing-div/')
-          const mapElement = MetadataUtils.getChildrenOrdered(
-            renderResult.getEditorState().editor.jsxMetadata,
-            renderResult.getEditorState().editor.elementPathTree,
-            mapParent,
-          )[0]
+        const mapParent = EP.fromString('utopia-storyboard-uid/scene-aaa/containing-div/')
+        const mapElement = MetadataUtils.getChildrenOrdered(
+          renderResult.getEditorState().editor.jsxMetadata,
+          renderResult.getEditorState().editor.elementPathTree,
+          mapParent,
+        )[0]
 
-          const counterTestId = getMapCounterTestId(mapElement.elementPath)
-          const counter = await renderResult.renderedDOM.findByTestId(counterTestId)
+        const counterTestId = getMapCounterTestId(mapElement.elementPath)
+        const counter = await renderResult.renderedDOM.findByTestId(counterTestId)
 
-          expect(counter.textContent).toEqual(overrideCount.toString())
-        })
+        expect(counter.textContent).toEqual(overrideCount.toString())
+        expect(counter.style.color).toEqual('var(--utopitheme-brandNeonPink)')
+        if (overrideSuccess) {
+          // successful override background color
+          expect(counter.style.backgroundColor).toEqual('var(--utopitheme-pinkSubdued)')
+        } else {
+          // failed override shows a diagonal strikethrough implemented with linear-gradient
+          expect(counter.style.background.slice(0, 15)).toEqual('linear-gradient')
+        }
```

Comment:
Hmmm testing the style isn't ideal 😬 Maybe it would be better to add a `data-*` attribute to the element that states whether it is successful or not?